sample-code/spring-app/src/test/java/com/sap/ai/sdk/app/controllers/OrchestrationTest.java

```diff
@@ -25,7 +27,12 @@ void testCompletion() {
     final var result = controller.completion();
 
     assertThat(result).isNotNull();
-    assertThat(result.getOrchestrationResult().getChoices().get(0).getMessage().getContent())
+    assertThat(
+            ((LLMModuleResultSynchronous) result.getOrchestrationResult())
+                .getChoices()
+                .get(0)
+                .getMessage()
+                .getContent())
```

Comment:
(Question/Major)

If I understand correctly, the PR changes public API in a way, that requires users to cast a result object to a specific class, before being able to read the response content. Is that correct? Is that the expected result?

Reply:
Note: this will be fixed by either AI/ai-sdk-java-backlog#110 or https://github.tools.sap/AI/llm-orchestration/pull/753